src/Raven.Client/Documents/Commands/GetConflictsResult.cs

```diff
@@ -20,4 +22,20 @@ public class Conflict
             public BlittableJsonReaderObject Doc { get; set; }
         }
     }
+
+    public class GetConflictsResultByEtag
```

Comment:
This is not longer by etag right? it is more `GetConflictsPreviewResult`

Reply:
done